### PR TITLE
Update Token.php

### PR DESCRIPTION
**Pull request by Lablnet**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | yes
| New feature?  | yes
| Deprecations? | no
| Tests pass?   | No
| Fixed Issue |  https://github.com/dframe/dframe/issues/46
| License       | MIT

### CODE REVIEW

File: src/Token.php [hunk 1]
```diff
@@ -68,27 +68,37 @@ public function clear()
...
 
-    /**
-     * @param      $values
-     * @param null $ttl
-     */
-    public function setMultiple($values, $ttl = null)
+        return $cache;
+    }
+	/**
+	/*Format like: $arrays = [
```
Comment (dusta) [inline src/Token.php]: The comment is incorrect.

File: src/Token.php [hunk 2]
```diff
@@ -68,27 +68,37 @@ public function clear()
...
+            ['key' => 'name','value' => 'SomeName'],
+			.....*/
+			
+	*/
+    public function setMultiple($caches, $ttl = null)
```
Comment (dusta) [inline src/Token.php]: Argument should be named $values 
like here: https://www.php-fig.org/psr/psr-16/

File: src/Token.php (hunk 1 above)
Comment (Lablnet) [reply src/Token.php]: you can correct that bro.

Comment (dusta) [reply src/Token.php]: Yes I can, but in code review it's not the point to fix code for someone else but for give feedback.

Comment (Lablnet) [reply src/Token.php]: i am unable to edit this commit again because i had deleted the forked repo.

**Comment by dusta** [comment]
#101

**Pull request closed without merging**